internal/services/databricks/databricks_workspace_resource.go

```diff
@@ -1099,3 +1180,84 @@ func workspaceCustomParametersString() []string {
 		"custom_parameters.0.vnet_address_prefix",
 	}
 }
+
+func flattenWorkspaceEnhancedSecurity(input *workspaces.EnhancedSecurityComplianceDefinition) []interface{} {
+	if input == nil {
+		return nil
+	}
+
+	enhancedSecurityCompliance := make(map[string]interface{})
+
+	if v := input.AutomaticClusterUpdate; v != nil {
+		enhancedSecurityCompliance["automatic_cluster_update_enabled"] = *v.Value != workspaces.AutomaticClusterUpdateValueDisabled
+	}
+
+	if v := input.EnhancedSecurityMonitoring; v != nil {
+		enhancedSecurityCompliance["enhanced_security_monitoring_enabled"] = *v.Value != workspaces.EnhancedSecurityMonitoringValueDisabled
+	}
+
+	if v := input.ComplianceSecurityProfile; v != nil {
+		enhancedSecurityCompliance["compliance_security_profile_enabled"] = *v.Value != workspaces.ComplianceSecurityProfileValueDisabled
```

Comment:
```suggestion
		enhancedSecurityCompliance["enhanced_security_monitoring_enabled"] = pointer.From(v.Value) != workspaces.EnhancedSecurityMonitoringValueDisabled
	}

	if v := input.ComplianceSecurityProfile; v != nil {
		enhancedSecurityCompliance["compliance_security_profile_enabled"] = pointer.From(v.Value) != workspaces.ComplianceSecurityProfileValueDisabled
```

Reply:
Fixed in 0159fa6fbef247bf18d40d1a9c9ec80247582a46